docker/manage

```diff
@@ -46,7 +46,7 @@ exportEnvironment() {
   export GENESIS_URL=${GENESIS_URL:-http://$DOCKERHOST:9000/genesis}
   export STORAGE_PATH=${STORAGE_PATH:-/tmp/tails-files}
   export LOG_LEVEL=${LOG_LEVEL:-INFO}
-  export TAILS_SERVER_URL=${TAILS_SERVER_URL:-http://$DOCKERHOST:6543}
+  export TAILS_SERVER_URL=${TAILS_SERVER_URL:-http://host.docker.internal:6543}
```

Comment:
We can probably not include this change.

Reply:
This change may not work on Linux.  I'm not sure if the Linux version of Docker has caught up with the Windows and Mac version that use `host.docker.internal`.  This was originally setup to autodetect the docker host URL or IP based on the platform.  Refer to the script at the top of the file.

If the Linux version of Docker has caught up and is `host.docker.internal` aware, you can ignore this comment.

Reply:
Looking through the docker release notes, it appears using `host.docker.internal` still requires some manual steps; it's not automatic like on Mac and Windows.

Reply:
I’ve reverted this change in the next PR as I hit this problem in testing.